pom.xml

```diff
@@ -14,10 +14,10 @@
     <properties>
         <revision>0.2</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.138.2</jenkins.version>
+        <jenkins.version>2.138.4</jenkins.version>
         <java.level>8</java.level>
         <useBeta>true</useBeta>
-        <workflow-api-plugin.version>2.32</workflow-api-plugin.version>
+        <workflow-api-plugin.version>2.34-SNAPSHOT</workflow-api-plugin.version> <!-- TODO https://github.com/jenkinsci/workflow-api-plugin/pull/88 -->
```

Comment:
Have not bothered to wait for an incremental deployment of this since I would rather clear up the timestamped snapshot upstream first.

Reply:
failure apparently because this SNAPSHOT wasn't found, did you deploy it in the meantime?

Reply:
No, I was just going to see if I could get approval on the upstream-upstream PRs so I could just use release versions in https://github.com/jenkinsci/workflow-api-plugin/pull/88 and _then_ switch this to an incremental dep. The CI is not valuable on this repo because the interesting test (the one being patched here) runs only with an AWS account.